### PR TITLE
Create Animal Table -Vet Clinic Database

### PR DESCRIPTION
**Pull request by harshi0102**
I created a table called animals in the vet clinic database.
Also, I created a table named animals with the following columns:
- id: integer
- name: string
- date_of_birth: date
- escape_attempts: integer
- neutered: boolean
- weight_kg: decimal

 I also inserted the required data and perform some Query operations as shown in the screenshots below

image
image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,8 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES(1, 'Agumon', '2020-02-03', 0, true, 10.23),
```
Comment (mukhammadrizooff) [inline data.sql]: - [ ] [REQUIRED] Good job so far! Please remove the `id` from your `data` 🙏🏼 

- `[HINT]`

```sql
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23');
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Gabumon', '2018/11/15', '2', 'true', '8.0');
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Pikachu', '2021/01/07', '1', 'false', '15.04');
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Devimon', '2017/05/12', '5', 'true', '11.0');
```

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @harshi0102, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by harshi0102** [comment]
According to the feedback of removing ID from data.sql, I removed that now from code written in sql.

**Review by mikethreels — APPROVED** [review]
Hi Harishika, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good job implementing the requested changes.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**